src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -170,6 +172,7 @@ func newBootstrapCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 	cmd.Flags().StringVar(&opts.address, "address", "", "microcluster address or CIDR, defaults to the node IP address")
 	cmd.Flags().StringVar(&opts.outputFormat, "output-format", "plain", "set the output format to one of plain, json or yaml")
 	cmd.Flags().DurationVar(&opts.timeout, "timeout", 90*time.Second, "the max time to wait for the command to execute")
+	cmd.Flags().StringVar(&opts.containerdBaseDir, "containerd-base-dir", "", "set the containerd base containerd directory")
```

Comment:
Has there been agreement from higher-ups that this should be a CLI flag?

Reply:
Done, went with the bootstrap config file option.